csharp/ql/lib/semmle/code/csharp/Type.qll

```diff
@@ -48,6 +48,11 @@ class Type extends Member, TypeContainer, @type {
 
   /** Holds if this type is a value type, or a type parameter that is a value type. */
   predicate isValueType() { none() }
+
+  /**
+   * Holds if this type is a ref like type.
```

Comment:
I think this comment deserves to be elaborated; right now it simply means that is is a `ref struct`, right?

Reply:
Yes, that is correct.
I will elaborate.